docs/how-to/accessibility.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Accessibility
+---
+
+# Accessibility
+
+Accessibility in a React Router app looks a lot like accessibility on the web in general. Using proper semantic markup and following the [Web Content Accessibility Guidelines (WCAG)][wcag] will get you most of the way there.
+
+React Router makes certain accessibility practices the default where possible and provides APIs to help where it's not. We are actively exploring and developing new APIs to make this even easier in the future.
```

Comment:
```suggestion
React Router makes certain accessibility practices the default where possible and provides APIs to help where it's not.
```

I'm not entirely sure what this was referencing at the time, and either we've fulfilled that promise or decided it didn't make sense. Either way, I'd prefer us no longer make this promise so open ended when there aren't current plans to add anything more around accessibility than what we've already added